### ISSUE TITLE
Remove to_device

### ISSUE THREAD

**Issue by canergen** [YosefLab/mrvi#85]
@martinkim0 Is this handled now after adding scvi-tools>1.0.0 support?
https://github.com/YosefLab/scvi-v2/blob/7f4984a8371c0ee43d60487bb280322dcd9a50f5/src/scvi_v2/_model.py#L134

**Comment by martinkim0** [comment]
I think so now that we have #82 merged, although that method isn't doing anything (?).

**Comment by canergen** [comment]
Yes, it was a placeholder. I remember to_device wasn't set up in Jax before we added the arguments to scvi-tools.
